tests/kafkatest/services/verifiable_producer.py

```diff
@@ -73,24 +73,27 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
         self.produced_count = {}
-        self.prop_file = ""
+
+    def prop_file(self, idx):
```

Comment:
also since verifiable producer support multiple producers, it could be useful in the future for example to set client ID based on index, or possibly make other configs different for different producers.